proto/spire/api/server/local_authority/v1/local_authority.proto

```diff
@@ -0,0 +1,174 @@
+syntax = "proto3";
+package spire.api.server.local_authority.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/server/local_authority/v1;local_authorityv1";
+
+// The LocalAuthority service provides a way to manage the signing keys (and
+// related material) of the SPIRE Server exposing it.
+service LocalAuthority {
+        // GetJWTAuthorityState returns the state of all locally configured
+        // JWT authorities.
+        rpc GetJWTAuthorityState(GetJWTAuthorityStateRequest) returns (GetJWTAuthorityStateResponse);
+
+        // PrepareJWTAuthority prepares a new JWT authority for use by
+        // generating a new key and injecting it into the bundle. This action
+        // will propagate the new public key cluster-wide.
+        rpc PrepareJWTAuthority(PrepareJWTAuthorityRequest) returns (PrepareJWTAuthorityResponse);
+
+        // ActivateJWTAuthority activates a prepared JWT authority for use,
+        // which will cause it to be used for all JWT signing operations
+        // serviced by this server going forward. If a new JWT authority has
+        // not already been prepared, a FailedPrecondition error will be returned.
+        rpc ActivateJWTAuthority(ActivateJWTAuthorityRequest) returns (ActivateJWTAuthorityResponse);
```

Comment:
if I follow you here we have 3 functions that allow rotation:
- Activate: it is like regular (and automatic) rotation
- Taint: rotate actual active authority and force mint of new SVIDs on agents, but bundle is not removed
- Revoke: rotate, and DELETE bundle, and force mint new SVIDs.

I think taint/revoke are good for different scenarios where we want to force a rotation, but when `Activate` is used? 
for testing? it will not force SVID rotations, why not to wait for regular rotation?